src/assets/styles/mixins.less

```diff
@@ -31,30 +31,30 @@
   white-space: nowrap;
 }
 
-.bubble-head(@top: auto, @bottom: auto, @left: auto, @right: auto) {
-  position: absolute;
+.bubble-head(@top: auto, @right: auto, @bottom: auto, @left: auto) {
+  .position(absolute, @top, @right, @bottom, @left);
```

Comment:
Not sure if I understand -- it's like any other helper function. There are 4 lines below that are removed with the use of the new mixin here.